CONTRIBUTING.md

```diff
@@ -228,32 +229,40 @@ only pull requests targeted directly against master.
 - Tag the release (use `git tag -a`) and create a release in Github
 - Delete the `RC` branches
 
-### Point Release Procedure
+### Stable Release Update Procedure
 
 At the moment, only a single major release will be supported, so all point releases will be based
 off of that release.
 
 In order to alleviate the burden for a single person to have to cherry-pick and handle merge conflicts
-of all desired backporting PRs to a point release, we instead maintain a living backport branch, where
+of all desired backporting PRs to a point release, we instead maintain a living SRU branch, where
 all desired features and bug fixes are merged into as separate PRs.
 
 Example:
 
-Current release is `v0.38.4`. We then maintain a (living) branch `backport/release/v0.38.x`. As PRs
-are merged into master, if they wish to be backported into the next `v0.38.x` point release, the
-author must:
-
-1. Add `backport` label
-2. Pull latest changes on the desired `backport/release/v*.*.x` branch
-3. Create a 2nd PR merging the respective backport PR into `backport/release/v*.*.x`
-
-This means it is the authors responsibility to fix any merge conflicts, update changelog entries, and
-ensure CI passes. If a PR originates from an external contributor, it may be a team members responsibility
-to perform this process instead of the original author.
+Current release is `v0.38.4`. We then maintain a (living) branch `sru/release/v0.38.N`, given N as
+the next patch release number (currently `0.38.5`) for the `0.38` release series. As bugs are fixed
+and PRs are merged into `master`, if a contributor wishes the PR to be released as SRU into the
+`v0.38.N` point release, the contributor must:
+
+1. Add `0.38-sru` label
```

Comment:
i would prefer to keep `backport` this is another thing that we will have to update on every major release. This increases the maintenance load and cognitive load of everything that needs to be updated in these scenarios

Reply:
Alright, so instead of `backport` I'd propose to use a semantic versioning-like nomenclature: `MAJOR.MINOR.PATCH`

Reply:
The thing is, regardless of the name (we agreed that we stick with `backport`), I think that such living branch should carry the next version number to 1. avoid versioning mistakes 2. make clear to users what changes are going to be in exactly which release. Wdyt? @marbar3778 @alexanderbez

Reply:
yes agree with you. `0.38-backport` works for me

Reply:
I actually meant "it should include the patch number" :) e.g. `0.38.5-backport`

Reply:
The proposal of `0.38.5-backport` looks fine to me. I am infavor of calling it backport and not changing historical nomenclature.

@alessio  Can you update this file accordingly?

Reply:
Fixed. Please check